src/app/dialogs/dialog-loan-lend/dialog-loan-lend.component.scss

```diff
@@ -1,397 +1,124 @@
-@import './../../../scss/mixins';
-@import './../../../scss/fonts';
-@import './../dialog-generic';
+@import "./../../../scss/mixins";
+@import "./../../../scss/fonts";
+@import "./../dialog-generic";
 
-.dialog-content {
-  .item {
-    &__select-label {
-      @include styled-font('Roboto', 400, 12px);
-      color: #989898;
-    }
-    &__select-value {
-      @include typography-headline(6);
-    }
-  }
-  .accordion-details {
-    mat-expansion-panel {
-      margin-top: 35px;
-    }
-  }
-}
-
-//Dialog Client
 .dialog {
-    max-width: 100vw;
-    min-height: fit-content;
-    overflow-x: hidden;
-}
-
-.mat-option {
-  height: auto;
-}
-.dialog-padding{
-  padding: 25px 15px;
-  text-align: left;
-}
+  max-width: 100vw;
+  min-height: fit-content;
+  overflow-x: hidden;
 
-.no-pad-left {
-  padding-left: 0;
-}
-
-.padding {
-    padding: 0px 20px;
-}
-
-.padding-2 {
-  padding: 0px 30px;
-}
-
-// ---------------
-
-.i {
-    display: inline-block;
-    cursor: pointer;
-    .fa-question-circle {
-        color: #6D6E6F !important;
-        margin: 0 8px;
+  .dialog-content {
+    .item {
+      &__select-label {
+        @include styled-font("Roboto", 400, 14px);
```

Comment:
@martinbobbio use @typography-body(3) instead of custom styled-font

Reply:
Implemented